c++/src/capnp/rpc.capnp

```diff
@@ -766,6 +766,10 @@ struct Disembargo {
   using EmbargoId = UInt32;
   # Used in `senderLoopback` and `receiverLoopback`, below.
 
+  using ThirdPartyEmbargoId = Data;
```

Comment:
Why not `AnyPointer` that's a network type parameter?

Reply:
Because the RPC system needs to create a HashMap keyed by this. Technically it could use an AnyPointer and canonicalize it for this purpose but that's a lot of overhead and I can't think of a reason why anyone would use anything other than random bytes here.